### DISCUSSION TITLE
I cant find the option to change Monika's hairstyle

### DISCUSSION THREAD

**Discussion by PolterGhosty** [Monika-After-Story/MonikaModDev#9243] [QA - tech support + general]
its been like this for **6 months** and **my affection is 700+**, i waited for the topic but i didn't came
maybe one of the old Submods i added  interfered with Monika having that topic where she can change her hiar.
The Submod is about Monika Automatically letting her hair down when its night time .
How can i fix this?

**Comment by Booplicate** [comment]
Are you using a translation? Can list all your submods?

**Reply by PolterGhosty** [reply]
here
i removed the submod that automatically lets her hair down 4 months ago
![image](https://user-images.githubusercontent.com/100363965/174449876-7c1354c6-e223-4ae5-848d-3be64c09dc0c.png)
![image](https://user-images.githubusercontent.com/100363965/174449886-cf515c66-5966-4532-b6d1-245f595b94f7.png)
![image](https://user-images.githubusercontent.com/100363965/174449904-357fe818-a197-4a3a-893a-254ee7fcdf94.png)
![image](https://user-images.githubusercontent.com/100363965/174449912-3720bfc1-47ca-4db7-bdcf-d43b7b62ff6c.png)
and this is the auto hair/clothes submod that i removed
https://github.com/multimokia/MAS-Submod-Auto-Outfit-Change
i hope you can help me
